### PR TITLE
added details mr joestar

### PR DESCRIPTION
**Pull request by samromer**
Yep that's a ded

### CODE REVIEW

File: resources/views/attractieDetail.blade.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <link href="{{ asset('../resources/css/attractieDetail.css') }}" rel="stylesheet">
+        @vite(['resources/css/contact.css', 'resources/js/app.js'])
+    <link
+      rel="icon"
+      href="https://media.tenor.com/j6HNDMU_fF4AAAAM/cow-dancing.gif"
+    />
+@endsection
+
+@section("title")
+    U did it gamer
```
Comment (JuliusDeBoer) [inline resources/views/attractieDetail.blade.php:14]: Is that the final title?

File: app/Http/Controllers/AttractieDetailController.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(Request $request)
```
Comment (JuliusDeBoer) [inline app/Http/Controllers/AttractieDetailController.php:29]: Would it be better to delete these unused functions?

File: resources/css/layout.css [hunk 3]
```diff
@@ -73,7 +73,7 @@ body{
     border-left: 8px solid rgb(132, 132, 132);
     border-right: 8px solid rgb(132, 132, 132);
 
-    background-image: url("https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2F64.media.tumblr.com%2F4f41d96967694aae9f7585e7efd1e8ca%2Ff40442872ae8b83c-37%2Fs400x600%2F78b368813035750a0ac48840aec0a9aafbd00581.gifv&f=1&nofb=1&ipt=a155bc76a66686e6fec1646bb2465d4d6634585c34f3a9b42ccf013123ef5960&ipo=images");
+    /* background-image: url("https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2F64.media.tumblr.com%2F4f41d96967694aae9f7585e7efd1e8ca%2Ff40442872ae8b83c-37%2Fs400x600%2F78b368813035750a0ac48840aec0a9aafbd00581.gifv&f=1&nofb=1&ipt=a155bc76a66686e6fec1646bb2465d4d6634585c34f3a9b42ccf013123ef5960&ipo=images"); */
```
Comment (JuliusDeBoer) [inline resources/css/layout.css:76]: Would it be cleaner if this line was deleted or uncommented?

**Review by JurreSerrarens — APPROVED** [review]
JohnApproved

**Pull request merged**